source/guides/guides/parallelization.md

```diff
@@ -151,18 +153,26 @@ For multiple runs to be grouped into a single run, it is required for CI machine
 
 You can test your application against different browsers and view the results under a single run within the Dashboard. Below, we simple name our groups the same name as the browser being tested:
 
-- The first group can be called `electron`. *Electron is the default browser used in Cypress runs*.
+- The first group can be called `Windows/Chrome 69`. 
+
+  ```shell
+  cypress run --record --group Windows/Chrome-69
+  ```
+
+- The second group can be called `Mac/Chrome 70`. 
 
   ```shell
-  cypress run --record --group electron
+  cypress run --record --group chrome --browser Mac/Chrome-70
```

Comment:
Let's make sure we don't miss up the arguments here. The arguments passed to the flag will be the one immediately following the flag. So, how this is written above actually says:

- group = 'chrome'
- browser = 'Mac/Chrome-70'

These arguments need to be switched around. The `--browser` flag is what's used to define the browser we want to run Cypress in and the `--group` flag is where we pass what we want to name our group.